datasets.md

```diff
@@ -0,0 +1,87 @@
+  2. Rename `ILSVRC2017_val.txt` to `val15.txt`
+
+### Structure of directory with dataset
+
+This section describes what structure of directory dataset must have to run the Accuracy Checker tool.
+* `<DATASET_DIR>` - root directory
+    * `ILSVRC2012_img_val` - directory containing the ILSVRC 2012 validation images
+    * `val.txt` - annotation file used for ILSVRC 2012
+    * `val15.txt` - annotation file used for ILSVRC 2015
+
+### Datasets in dataset_definitions.yml
+* `imagenet_1000_classes` used for evaluation models trained on ILSVRC 2012 dataset with 1000 classes. (model examples: [alexnet](models/public/alexnet/alexnet.md), [vgg16](models/public/vgg16/vgg16.md))
+* `imagenet_1000_classes_2015` used for evaluation models trained on ILSVRC 2015 dataset with 1000 classes. (model examples: [se-resnet-152](models/public/se-resnet-152/se-resnet-152.md), [se-resnext-50](models/public/se-resnext-50/se-resnext-50.md))
+* `imagenet_1001_classes` used for evaluation models trained on ILSVRC 2012 dataset with 1001 classes (background label + original labels). (model examples: [googlenet-v2-tf](models/public/googlenet-v2-tf/googlenet-v2-tf.md), [resnet-50-tf](models/public/resnet-50-tf/resnet-50-tf.md))
+
+## [Common Objects in Context (COCO)](http://cocodataset.org/#home)
+
+### How download dataset
+
+To download COCO dataset, you need to follow the steps below:
+1. Go to the [COCO](http://cocodataset.org/#home) website
+2. Go to the `Dataset-> Download` tab
+3. Select [`2017 Val images`](http://images.cocodataset.org/zips/val2017.zip) and [`2017 Train/Val annotations`](http://images.cocodataset.org/annotations/annotations_trainval2017.zip) and download these archives
+4. Unpack archives
+
+### Structure of directory with dataset
+
+This section describes what structure of directory dataset must have to run the Accuracy Checker tool.
+* `<DATASET_DIR>` - root directory
+    * `val2017` - directory containing the COCO 2017 validation images
+    * `instances_val2017.json` - annotation file which used for object detection and instance segmentation tasks
+    * `person_keypoints_val2017.json` - annotation file which used for human pose estimation tasks
+
+### Datasets in dataset_definitions.yml
+* `ms_coco_mask_rcnn` used for evaluation models trained on COCO dataset for object detection and instance segmentation tasks. Background label + label map with 80 public available object categories are used. Annotations are saved in image id ascend order.
+* `ms_coco_detection_91_classes` used for evaluation models trained on COCO dataset for object detection tasks. Background label + original label map with 91 object categories are used (public available 80 categories). Annotations are saved in image id ascend order. (model examples: [faster_rcnn_resnet50_coco](models/public/faster_rcnn_resnet50_coco/faster_rcnn_resnet50_coco.md), [ssd_resnet50_v1_fpn_coco](models/public/ssd_resnet50_v1_fpn_coco/ssd_resnet50_v1_fpn_coco.md))
```

Comment:
This seems incomplete/contradictory. Are there 91 categories or 80?

Reply:
Description has been updated, please re-check